conans/test/integration/toolchains/gnu/test_pkgconfigdeps.py

```diff
@@ -460,6 +462,8 @@ def package_info(self):
     assert "Requires: compo1" == get_requires_from_content(pc_content)
 
 
+# FIXME: Are they really useful all those warning messages?
+@pytest.mark.skip
```

Comment:
If you are removing the warnings, shouldn't you remove the test as well instead of skipping it?

Reply:
Yes, sure. I did not do that till knowing your opinion 😉